src/main/res/layout/layout_authentication.xml

```diff
@@ -18,101 +17,119 @@
-
-		<Space
-			android:id="@+id/view_message_empty"
-			android:layout_width="0dp"
-			android:layout_height="0dp"/>
-
-		<TextView
-			android:id="@+id/text_message_authentication"
-			android:text="@string/message_error_authentication"
-			android:gravity="center_horizontal"
-			android:layout_width="match_parent"
-			android:layout_height="wrap_content"/>
-
-		<TextView
-			android:id="@+id/text_message_authentication_connection"
-			android:text="@string/message_error_authentication_connection"
-			android:gravity="center_horizontal"
-			android:layout_width="match_parent"
-			android:layout_height="wrap_content"/>
-
-		<TextView
-			android:id="@+id/text_message_authentication_empty"
-			android:text="@string/message_error_authentication_field_empty"
-			android:gravity="center_horizontal"
-			android:layout_width="match_parent"
-			android:layout_height="wrap_content"/>
-
-	</ViewAnimator>
-
-</LinearLayout>
\ No newline at end of file
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent">
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:descendantFocusability="beforeDescendants"
+        android:focusableInTouchMode="true"
+        android:orientation="vertical"
+        android:paddingLeft="32dp"
+        android:paddingRight="32dp">
+
+        <ImageView
+            android:id="@+id/icon"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingLeft="32dp"
+            android:paddingRight="32dp"
+            android:src="@drawable/ic_banner" />
+
+        <Space
+            android:layout_width="match_parent"
+            android:layout_height="8dp" />
+
+        <android.support.design.widget.TextInputLayout
+
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:theme="@style/Theme.AppCompat"
+            app:passwordToggleEnabled="true">
+
+            <EditText
+                android:id="@+id/edit_username"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="@string/hint_username"
+                android:singleLine="true" />
+
+        </android.support.design.widget.TextInputLayout>
+
+        <Space
+            android:layout_width="match_parent"
+            android:layout_height="8dp" />
+
+        <android.support.design.widget.TextInputLayout
+
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:theme="@style/Theme.AppCompat"
+            app:passwordToggleEnabled="true">
+
+            <EditText
```

Comment:
@hshrimali Why have you changed it to `EditText`? There's a slight difference between using `TextInputEditText` and `EditText`. Pls read this stackoverflow [answer](http://stackoverflow.com/a/36269036/6352482).

Anyway I have implemented this in another pr #90 resolving bug #87.